cmd/buildah/run.go

```diff
@@ -51,6 +51,9 @@ func runCmd(c *cli.Context) error {
 	}
 	name := args[0]
 	args = args.Tail()
+	if len(args) > 0 && args[0] == "--" {
```

Comment:
Is the '--' standard?  I'm not seeing this in docker run.  gdb does --args, I'd prefer that as it's clearer and easier to search in a man page.